addons/l10n_dk_oioubl/data/oioubl_201_templates.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <template id="oioubl_201_AddressType" inherit_id="account_edi_ubl_cii.ubl_20_AddressType" primary="True">
+        <xpath expr="//*[local-name()='StreetName']" position="before">
+            <t xmlns="urn:oasis:names:specification:ubl:schema:xsd:Invoice-2"
+               xmlns:cac="urn:oasis:names:specification:ubl:schema:xsd:CommonAggregateComponents-2"
+               xmlns:cbc="urn:oasis:names:specification:ubl:schema:xsd:CommonBasicComponents-2">
+                <cbc:AddressFormatCode t-att="vals.get('address_format_code_attrs', {})" t-out="vals.get('address_format_code')"/>
```

Comment:
It seems to be standard UBL 2.0 (http://www.datypic.com/sc/ubl20/e-cac_PostalAddress.html). Why not adding it in `account_edi_ubl_cii` ?